src/Nancy.Swagger/SwaggerConfig.cs

```diff
@@ -8,8 +10,8 @@ public static class SwaggerConfig
         /// <summary>
         /// Contains all resolved ids by the <see cref="DefaultModelIdConvention"/>.
         /// </summary>
-        private static ConcurrentDictionary<string, Type> _resolvedModelIds = new ConcurrentDictionary<string, Type>();        
-        
+        private static ConcurrentDictionary<string, Type> _resolvedModelIds = new ConcurrentDictionary<string, Type>();
+
         /// <summary>
         /// The default resource listing path, <c>api-docs</c>.
```

Comment:
In swagger 2.0 default path should be swagger.json, see https://github.com/swagger-api/swagger-spec/wiki/Swagger-1.2-to-2.0-Migration-Guide#whats-changed